src/common/utils/test.util.ts

```diff
@@ -56,3 +56,21 @@ export async function createTestUser({
 
   return { ...result.user, session: result.session.id };
 }
+
+export async function createTestCategory({
```

Comment:
My reason to create a separate testUser was so that I can pass a role because normally for register route we cannot pass role, what is the purpose of using this over normal repository?

Reply:
You are right, i could have used directly normal repository. Its done.